build/vagrant/Makefile

```diff
@@ -1,47 +1,63 @@
 SHELL := /bin/bash
-SOURCE_CIDR := $(shell curl -sSL 'https://api.ipify.org')/32
+
+PACKER_LOG = '1'
 
 PACKER_VERSION ?= $(shell grep 'FROM hashicorp/packer' Dockerfile 2> /dev/null | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' 2> /dev/null)
 PACKER_URL ?= https://releases.hashicorp.com/packer/$(PACKER_VERSION)/packer_$(PACKER_VERSION)_linux_amd64.zip
-TERRAFORM_VERSION ?= $(shell grep 'FROM hashicorp/terraform' Dockerfile 2> /dev/null | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' 2> /dev/null)
-TERRAFORM_URL ?= https://releases.hashicorp.com/terraform/$(TERRAFORM_VERSION)/terraform_$(TERRAFORM_VERSION)_linux_amd64.zip
 
 ISO_URL_CENTOS7_BASE ?= http://mirror.cs.vt.edu/pub/CentOS/7/isos/x86_64
 ISO_URL_CENTOS7_FILTER ?= grep -Po '(?<=href=")CentOS-7-x86_64-Minimal-.*\.iso(?=")'
-ISO_URL_CENTOS7 ?= $(ISO_URL_CENTOS7_BASE)/$(shell curl -sSL $(ISO_URL_CENTOS7_BASE) | $(ISO_URL_CENTOS7_FILTER))
+
+CODEBUILD_SOURCE_REPO_URL ?= https://github.com/plus3it/spel.git
```

Comment:
Yes, since the `?=` means you can override it from the cli, but doesn't need to be since we can't build vagrant boxes with this template anywhere else, due to the requirement for metal ec2 build instances.

Reply:
Yeah. Mostly "readying for mists of the future" question.